examples/README.md

```diff
@@ -0,0 +1,49 @@
+Creating a basic S2I builder image  
+---
+
+### Getting started  
+
+Directory skeleton:  
+
+* `Dockerfile` – standard Dockerfile where we’ll define the base builder image.  
+* `assemble` - responsible for building the application.  
+* `run` - responsible for running the application.  
+* `save-artifacts` - optional script for incremental builds that save built artifacts.  
+* `usage` - optional script responsible for printing the usage of the builder.  
+
+The first step that we need to do is create a `Dockerfile` that installs all the necessary tools and libraries that are needed to build and run our application.  
+You can find an example of the Dockerfile [`here`](nginx-app/Dockerfile).   
+
+The next step is to create an `assemble` script that will install our application. This is where we build python modules or bundle install our required components. We can also restore any saved artifacts from the previous image. In our example that we have [`here`](nginx-app/assemble) it will only copy our index.html over the default one.  
+
+Now we can create our `run` script that will start the application. You can see an example  [`here`](nginx-app/run).
+
+
+Optionally we can also specify a `save-artifacts` script, which allows a new build to reuses content from a previous version of the application image.
+An example can be found [`here`](nginx-app/save-artifacts).  
+
+We can also inform the user on how to build an application image based on our builder image via the `usage` script. An example can be found [`here`](nginx-app/usage).
+You can print this output with `s2i usage nginx-centos7`.
```

Comment:
this line (26) needs to come after you've built the builder image.  you can't invoke s2i usage until you've built the image.